module/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "load_balancer_url" {
```

Comment:
Let's add the ids of all created entities, e.g. https://github.com/quintilesims/consul/blob/master/terraform/outputs.tf